crates/ide/src/status.rs

```diff
@@ -34,15 +34,24 @@ fn macro_syntax_tree_stats(db: &RootDatabase) -> SyntaxTreeStats {
 // image::https://user-images.githubusercontent.com/48062697/113065584-05f34500-91b1-11eb-98cc-5c196f76be7f.gif[]
 pub(crate) fn status(db: &RootDatabase, file_id: Option<FileId>) -> String {
     let mut buf = String::new();
+    let count = profile::countme::get_all();
     format_to!(buf, "{}\n", FileTextQuery.in_db(db).entries::<FilesStats>());
     format_to!(buf, "{}\n", LibrarySymbolsQuery.in_db(db).entries::<LibrarySymbolsStats>());
     format_to!(buf, "{}\n", syntax_tree_stats(db));
-    format_to!(buf, "{} (macros)\n", macro_syntax_tree_stats(db));
-    format_to!(buf, "{} total\n", memory_usage());
-    format_to!(buf, "\ncounts:\n{}", profile::countme::get_all());
+    format_to!(buf, "{} (Macros)\n", macro_syntax_tree_stats(db));
+    format_to!(buf, "{} in total\n", memory_usage());
+    format_to!(
```

Comment:
This would look nicer with the `if` pulled out of the macro call.

But maybe we should hide these if `RA_COUNT` is not set? The "all counts are zero" check doesn't really pull its weight.

Reply:
Not sure. Can you provide some more info? I'm confused on want you want. That code is just to make the messages have proper grammar.

Reply:
`countme` only works if you set the `RA_COUNT` environment variable (`env::var("RA_COUNT").is_ok()`), so it makes more sense to hide the value if counting is disabled.

Reply:
Okay, think I have resolved it in commit [`2e60908` (#9269)](https://github.com/rust-analyzer/rust-analyzer/pull/9269/commits/2e60908fde3000009973f56d6ebfc3ae4d9151de)